tools/loader.md

```diff
@@ -104,4 +114,5 @@ Or use configuration file "config.toml":
     
 ### Note
 
-If you use the default checkpoint file, after importing the data of a database, please delete loader.checkpoint before you begin to import the next database. When importing each database, explicitly specify the file name of checkpoint is recommended.
++ If you use the default checkpoint file, after importing the data of a database, please delete loader.checkpoint before you begin to import the next database. When importing each database, explicitly specify the file name of checkpoint is recommended.
```

Comment:
If you use the default checkpoint-schema, after importing the data of a database, please drop database tidb_loader before you begin to import the next database